Fixed navbar on top in Navbar ui

aryadevesh opened the pull request:
## 📋 Description

I changed the position of the navbar. Previously it was not fixed on top, Now I fixed on top. It improves user-experience.

## 🔨 Changes Made

Fixes #100 

- 
- 
- 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings.
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [ ] New feature (non-breaking change which adds functionality) ✨
- [x] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

## 🤝 Related Issues

<!-- List any issues related to this PR. Link them using "Fixes #issue-number" to automatically close them when the PR is merged. -->

- Fixes #100 
<img width="681" alt="image" src="https://github.com/user-attachments/assets/69d5daaa-1e00-4a5c-a7eb-eed03d02143a">

<!-- If your changes include any UI updates, add screenshots here to show the changes. -->



*Thank you for contributing to Truth-Tunnel! We look forward to reviewing your PR and getting it merged!*

MitulSonagara commented:
@aryadevesh 

![image](https://github.com/user-attachments/assets/abe2ce31-d340-41a6-a760-51d7f9e95665)

Please solve this ...


![image](https://github.com/user-attachments/assets/6a589c41-1c85-42b6-83fa-9a9124e8156c)
Also make it visible for light mode

aryadevesh commented:
ok! I shall do it!

aryadevesh commented:
Fixed :- 
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/5bae1c4a-69fc-4afa-b345-dee6751f8e72">

Fixed:-
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/1a645fdd-673d-480e-8599-c834ceabaf00">
